src/coreclr-embedding-test/src/ScriptingManager.cpp

```diff
@@ -0,0 +1,149 @@
+//
+//  ScriptingManager.cpp
+//  embedding_test
+//
+//  Created by Leonidas Neftali Gonzalez Campos on 06/12/23.
+//
+
+#include "ScriptingManager.hpp"
+
+#define DOTNET_CMD "hostfxr_initialize_for_dotnet_command_line"
```

Comment:
Consider using a constexpr value instead of a macro as they're part of the language.

A constexpr std::string_view would be a better choice.

Reply:
True, will add it as a constant expression